lib/net/ldap/pdu.rb

```diff
@@ -84,7 +84,8 @@ class Error < RuntimeError; end
 
   def initialize(ber_object)
     begin
-      @message_id = ber_object[0].to_i
+      id =  ber_object[0]
```

Comment:
what's the point of this 2 line change?

Reply:
No one, sorry, I try to understand the ber object and I forgot that lines